pkg/model/dev.go

```diff
@@ -38,7 +38,7 @@ import (
 
 var (
 	//OktetoBinImageTag image tag with okteto internal binaries
-	OktetoBinImageTag = "okteto/bin:1.3.4"
+	OktetoBinImageTag = "okteto/bin:1.3.5"
```

Comment:
@jLopezbarb did you build this image?

Reply:
Moved the image to repo https://github.com/okteto/bin-image